src/main/resources/assets/js/components/modals/AboutModalComponent.jsx

```diff
@@ -6,14 +6,6 @@ var BackboneMixin = require("../../mixins/BackboneMixin");
 var ModalComponent = require("../components/../ModalComponent");
```

Comment:
Isn't this a page now? I don't see the ModalComponent being used, and it seems the name + directories are wrong.